CONTRIBUTING.md

```diff
@@ -17,7 +17,7 @@ If you have any questions, please reach out on the [HoloLens forums](https://for
 For things like fixing typos and small bug fixes, you can skip this step.
 
 If your change is more than a simple fix, please don't just create a big pull request. 
-Instead, start by [opening an issue](https://github.com/Microsoft/HoloToolkit-Unity/issues) describing the problem you want to solve and how you plan to approach the problem. 
+Instead, start by [opening an issue](https://github.com/Microsoft/HoloToolkit-Unity/issues) describing the problem you want to solve and how you planned to approach the problem. 
```

Comment:
why past tense?

Reply:
Because what you had planned should be done before your action, am I right? Haha

Reply:
The idea here is that you should post your proposal before you do the work. That way other developers can comment on your plan and hopefully save you from potential pitfalls. So this is properly present tense.